src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -130,6 +130,7 @@ export default {
 		KvLoadingOverlay,
 		LendHeader,
 		AddToBasketInterstitial,
+		lendFilterExpVersion: '',
```

Comment:
This needs to be returned to the `data` object down below.

Reply:
You are right, I forgot to revert that part! 👍